obsah/__init__.py

```diff
@@ -414,7 +414,7 @@ def reset_args(application_config: ApplicationConfig, metadata: dict, args: argp
                     for arg in reset_values:
                         if arg in persist_params and persist_params[arg] == getattr(args, arg):
                             delattr(args, arg)
-            for reset_arg in getattr(args, 'obsah_reset', []):
+            for reset_arg in (getattr(args, 'obsah_reset', []) or []):
```

Comment:
Perhaps this is nicer?
```suggestion
            for reset_arg in (getattr(args, 'obsah_reset', None) or []):
```

Or have a condition around it?
```suggestion
            if hasattr(args, 'obsah_reset'):
                for reset_arg in args.obsah_reset:
```

You could even suppress the `AttributeError` here too.

Reply:
We had the `hasattr` condition before https://github.com/theforeman/obsah/commit/63aebd096813305361a6fc59b5ea284fa2bb6f37 :see_no_evil: